src/main/java/com/jhuguet/sb_taskv1/app/repositories/UserRepository.java

```diff
@@ -2,6 +2,10 @@
 
 import com.jhuguet.sb_taskv1.app.models.User;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface UserRepository extends JpaRepository<User, Integer> {
+    @Query(value = "SELECT * FROM USERS u WHERE u.username = ?1", nativeQuery = true)
```

Comment:
Why did you write native query here? Spring data will generate the query by itself if method name will match spring data convention

Reply:
Will delete nativeQuery statement